frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -485,11 +497,11 @@ export const useNotebookStatus = (
 
   // Parse the last event
   const lastItem = filteredEvents[filteredEvents.length - 1];
-  const { step, description, status } = getNotebookEventStatus(lastItem, gracePeriod);
+  const { step, status } = getNotebookEventStatus(lastItem, gracePeriod);
 
   return [
     {
-      currentEvent: step ? ProgressionStepTitles[step] : description ?? '',
+      currentEvent: ProgressionStepTitles[step],
```

Comment:
```suggestion
      currentEvent: description || ProgressionStepTitles[step],
```
Best have the descriptor over the step name